indico/modules/events/abstracts/operations.py

```diff
@@ -64,6 +79,19 @@ def create_abstract(event, abstract_data, custom_fields_data=None):
     return abstract
 
 
+def update_abstract(abstract, abstract_data, custom_fields_data=None):
+    tracks = abstract_data.pop('submitted_for_tracks', None)
+    if abstract.edit_track_mode == EditTrackMode.both:
```

Comment:
I don't think we need this check. Is there any case in which we should allow editing the abstract but not the tracks?